label_studio/core/settings/base.py

```diff
@@ -714,3 +714,6 @@ def collect_versions_dummy(**kwargs):
     CSP_INCLUDE_NONCE_IN = ['script-src', 'default-src']
 
     MIDDLEWARE.append('core.middleware.HumanSignalCspMiddleware')
+
+CLOUD_PAGE_CHECKED_OBJECTS = get_env('CLOUD_PAGE_CHECKED_OBJECTS', 10000)
```

Comment:
Since we're in a global config file, mind naming these something more specific? E.g. `DATA_DISCOVERY_NUM_OBJECTS_TO_CHECK_BEFORE_READ`? (I know this is pretty verbose, I'm up for some middle ground or other ideas, just want it to be relatively clear what these are for)